bom/application/pom.xml

```diff
@@ -15,6 +15,7 @@
 
     <properties>
         <bouncycastle.version>1.66</bouncycastle.version>
+        <bouncycastle.fips.version>1.0.2</bouncycastle.fips.version>
```

Comment:
Is this version somehow related to the `bouncycastle.version` above? If so, maybe it would be nice to add a comment to let maintainers know that this version needs to be updated if the version above changes

Reply:
Hi @gastaldi No, `bc-fips` repackages a good number of classes from the regular BC provider and adds some add ons